website/src/components/DownloadLink/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Link from "@docusaurus/Link";
+
+function SourceTarballUrl() {
+    const {
+        siteConfig: {customFields},
+    } = useDocusaurusContext();
+    const version = customFields.version;
+    // TODO: We can retrieve if staging as customFields.isStaging,
+    //  but which links should we use for staging?
+    const link = `https://www.apache.org/dyn/closer.lua/opendal/${version}/apache-opendal-core-${version}-src.tar.gz?action=download`;
+    return <Link to={link}>0.45.1</Link>
```

Comment:
To be determinate.

Reply:
How about referring to the release candidate?
😵But it will be different from the final release download link.

Reply:
My second comment is exactly what you said and I list the concern.

Reply:
According to @sebbasf on the mailing list:

> I think the staging download page should show the intended final link values. These will not point to an actual file (apart from KEYS), but it should still be possible to check that the links have the correct format.

I push https://github.com/apache/opendal/pull/4565/commits/caa45b1ac991b67beb9bcd11c74448e18ba23d42 to implement this manner.

I'll fix up other CI failure and then we can merge and start the release of 0.46.0 to test it out.